src/chi/components/card/card.scss

```diff
@@ -426,6 +426,56 @@
     }
   }
 
+  &.-alert {
+    box-shadow: $card-alert-box-shadow;
+
+    &.-empty {
+      border: 0.0625rem solid $card-empty-border-color;
+
+      .chi-accordion__item {
+        &.-expanded {
+          .chi-accordion__trigger {
+            border-bottom: 0.0625rem solid $card-empty-border-color;
+            padding: $card-alert-empty-accordion-trigger-padding;
+          }
+        }
+
+        .chi-accordion__content {
+          padding-bottom: $card-alert-empty-accordion-content-padding-y;
+          padding-top: $card-alert-empty-accordion-content-padding-y;
+        }
+
+        .chi-card__caption {
+          .chi-card__empty-content {
+            align-items: center;
+            display: flex;
+            flex-direction: column;
+            gap: $card-alert-empty-content-gap;
+
+            &-icon {
+              background-color: $color-grey-20;
```

Comment:
@ASanchLumen  could we add card-alert-empty... spefic vars here? Same for the props without specific vars.